workers/ohsome_quality_analyst/cli.py

```diff
@@ -214,18 +212,13 @@ def create_report(
                 force=force,
             )
         )
-        if outfile:
-            if fid_field is None:
-                fid_field = DATASETS[dataset_name]["default"]
-            # TODO: Sanity check or remove this line
-            feature = asyncio.run(
-                db_client.get_feature_from_db(dataset_name, feature_id, fid_field)
-            )
-            feature = update_features_report(feature, report)
-            write_geojson(outfile, feature)
-        else:
-            click.echo(report.metadata)
-            click.echo(report.result)
+        geojson_object = report.as_feature()
+    if outfile:
+        write_geojson(outfile, geojson_object)
+    else:
+        click.echo(
+            geojson.dumps(geojson_object, default=datetime_to_isostring_timestamp)
+        )
```

Comment:
is it good to have something like this dependent on the parameter `outfile`? Wouldn't it be better if I could have both outputs at the same time?

Reply:
Then the whole GeoJSON would be printed to stdout all the time.
We could also printout only the results not the geometry for example.

Not sure what the best approache here is.

What does the user expect? What is the most helpful data printed to stdout?

Reply:
I changed the behavior to always print to stdout and made the help text more explicit.

Regarding the output of a not so readable GeoJSON to stdout see issue: https://github.com/GIScience/ohsome-quality-analyst/issues/89

Reply:
thanks :))